include/evmc/loader.h

```diff
@@ -0,0 +1,24 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#pragma once
+
+#if __cplusplus
+extern "C" {
+#endif
+
+enum evmc_loader_error_code
+{
+    EVMC_ERRC_SUCCESS = 0,
+    EVMC_ERRC_CANNOT_OPEN,
+    EVMC_ERRC_SYMBOL_NOT_FOUND,
+    EVMC_ERRC_INVALID_ARGUMENT,
+};
+
+struct evmc_instance* evmc_load(const char* filename, enum evmc_loader_error_code* error_code);
```

Comment:
Should have documentation here. Also mention that `error_code` can be `NULL` and in that case it won't be set.

Reply:
Okay this was added with a subsequent commit.